libpod/container_internal.go

```diff
@@ -183,6 +183,8 @@ func newContainer(rspec *spec.Spec, lockDir string) (*Container, error) {
 	ctr := new(Container)
 	ctr.config = new(ContainerConfig)
 	ctr.state = new(containerState)
+	// Exit Code 125 indicates exit code was never set.
+	ctr.state.ExitCode = 125
```

Comment:
I still don't like this. A running container shouldn't show as errored.

Reply:
It should also not show that it succeeded.